Applications/Spreadsheet/SpreadsheetView.h

```diff
@@ -46,6 +46,9 @@ class SpreadsheetView final : public GUI::Widget {
     Function<void(const Position&, Cell&)> on_selection_changed;
     Function<void()> on_selection_dropped;
 
+    void switched_out();
+    void switched_in();
```

Comment:
Instead of these, you can override `Widget::hide_event()` and `Widget::show_event()` which will get delivered when the `TabWidget` switches between tabs :)